### PR TITLE
Activate Travis-CI

### PR DESCRIPTION
**Pull request by ocefpaf**
You will need to activate Travis-CI for your repository, but meanwhile you can see the results in from my fork at https://travis-ci.org/ocefpaf/skprocrustes/jobs/251116813

I will comment inline the changes.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+matrix:
+  fast_finish: true
+  include:
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+  allow_failures:
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
```
Comment (ocefpaf) [inline .travis.yml]: This `.travis.yml`  is mostly boilerplate. Please let me know if you have any specific question about it.
I only added to items to the test matrix, a coding standard and a "default" test.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+before_install:
+    - wget http://bit.ly/miniconda -O miniconda.sh
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - export PATH="$HOME/miniconda/bin:$PATH"
+    - conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    - conda update conda
+    - conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
+    - source activate TEST
```
Comment (ocefpaf) [inline .travis.yml]: I am using conda packages for the testing and the failures may be mismatches due to the underlying `blas`.
In this case I believe it is `mkl`. We can try `openblas` later.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+install:
+    - python setup.py sdist && version=$(python setup.py --version) && pushd dist  && pip install scikit-procrustes-${version}.tar.gz && popd
```
Comment (ocefpaf) [inline .travis.yml]: This tests the creation of the source distribution tarball.

File: requirements-dev.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+flake8
+pytest
+nose
```
Comment (ocefpaf) [inline requirements-dev.txt]: Consider dropping nose entirely. It is deprecate. (I can send a PR to adapt the test to be 100% `py.test`. The bolierplate reduction is worth it.)

File: setup.py [hunk 5]
```diff
@@ -1,87 +1,76 @@
 #!/usr/bin/env python
-descr = """\
```
Comment (ocefpaf) [inline setup.py]: Sorry... I know it looks like I defaced your `setup.py`, but I tried to make it a little simpler and removed the bogus dependency on `numpy` as install time.

File: setup.py [hunk 6]
```diff
@@ -1,87 +1,76 @@
...
+with open('requirements.txt') as f:
+    tests_require = f.readlines()
+install_requires = [t.strip() for t in tests_require]
 
-    # Avoid non-useful msg: "Ignoring attempt to set 'name' (from ... "
-    config.set_options(ignore_setup_xxx_py=True,
-                       assume_default_configuration=True,
-                       delegate_options_to_subpackages=True,
-                       quiet=True)
+descr = """\
```
Comment (ocefpaf) [inline setup.py]: This info could be in the README and we can read it from there.

File: skprocrustes/tests/test_skprocrustes.py [hunk 7]
```diff
@@ -4,17 +4,16 @@
                            assert_almost_equal, assert_warns, assert_array_less)
 from scipy import linalg as sp
 import skprocrustes as skp
-import nose
```
Comment (ocefpaf) [inline skprocrustes/tests/test_skprocrustes.py]: `nose` needs to be present but not is imported. The rest of the changes in this file are blank space noise. Sorry :grimacing:

File: skprocrustes/tests/test_version.py [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Example test file.
 
 from numpy.testing import assert_equal
-import skexample
```
Comment (ocefpaf) [inline skprocrustes/tests/test_version.py]: copy-n-pasta fix :wink:

**Comment by melissawm** [comment]
@ocefpaf thank you so much! I'm just blindly following what's listed in the scipy documentation but it seems that it is outdated some places. I'll research more about travis when I get back. The tests need to be rewritten because I had already noted the difference in end values, but I just needed to get it done ;) I'll add a bug about it. Thanks again :)

**Comment by ocefpaf** [comment]
>  I'll research more about travis when I get back.

Let me know if you have any questions. 

PS: the conda version of your package is already available at https://anaconda.org/conda-forge/scikit-procrustes/files

**Pull request merged**